### PR TITLE
Navigator: cookieEnabled warning regarding Safari is outdated #32845

### LINKED ISSUE #32845: Navigator: cookieEnabled warning regarding Safari is outdated
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/cookieEnabled

### What specific section or headline is this issue about?

The first note

### What information was incorrect, unhelpful, or incomplete?

`Note: When the browser is configured to block third-party cookies, and navigator.cookieEnabled is invoked inside a third-party iframe, it returns true in Safari, Edge Spartan and IE (while trying to set a cookie in such scenario would fail). It returns false in Firefox and Chromium-based browsers.`

### What did you expect to see?

Safari technical preview 188 and latest webkit update has fixed navigator.cookieEnabled for Safari returning the correct value.

### Do you have any supporting links, references, or citations?

https://developer.apple.com/documentation/safari-technology-preview-release-notes/stp-release-188
https://bugs.webkit.org/show_bug.cgi?id=267789

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by kaushik95**
### Description

Fixes issue #32845

### Motivation

I'm interested in solving issues.

### Related issues and pull requests

"Fixes #32845"

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Tagging Hamish since I think it's related to cookie privacy. Especially the bite below that talks about experimental Firefox—maybe these changes should be elaborated on?

**Comment by hamishwillee** [comment]
I'll look on Friday. Thanks @Josh-Cena @kaushik95

**Review by hamishwillee — CHANGES_REQUESTED** [review]
Thank you, but this is not the right fix - all this text should be removed and the information should be added to the compatibility data.
I am confirming the best way to do that and will update this afterwards.

File: files/en-us/web/api/navigator/cookieenabled/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The property is read-only.
 A boolean.
 
 > [!NOTE]
```
Comment (hamishwillee) [inline files/en-us/web/api/navigator/cookieenabled/index.md:19]:
```suggestion
```

File: files/en-us/web/api/navigator/cookieenabled/index.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The property is read-only.
...
-> When the browser is configured to block [third-party cookies](/en-US/docs/Web/Privacy/Third-party_cookies), and `navigator.cookieEnabled` is invoked inside a third-party `<iframe>`, it returns `true` in Safari and IE (while trying to set a cookie in such scenario would fail). It returns `false` in Firefox and Chromium-based browsers.
+> When the browser is configured to block [third-party cookies](/en-US/docs/Web/Privacy/Third-party_cookies), and `navigator.cookieEnabled` is invoked inside a third-party `<iframe>`, it returns `true` in IE (while trying to set a cookie in such scenario would fail). It returns `false` in Firefox, Safari and Chromium-based browsers.
```
Comment (hamishwillee) [inline files/en-us/web/api/navigator/cookieenabled/index.md:21]:
```suggestion
```

**Comment by hamishwillee** [comment]
@kaushik95 I've added two suggestions to remove the note altogether - can you please accept them (your github branch isn't set up to take changes)? 

Then we can merge this once https://github.com/mdn/browser-compat-data/pull/24402 goes in.

Note, we don't need the info about IE anymore, so once this is in the compatibility data, the whole note isn't needed.

**Review by hamishwillee — APPROVED** [review]
Thank you very much.

**Pull request merged**